src/com/magento/idea/magento2uct/ui/ConfigurationDialog.java

```diff
@@ -221,6 +230,21 @@ private void setDefaultValues() {
         }
         final Boolean shouldIgnore = settingsService.shouldIgnoreCurrentVersion();
         ignoreCurrentVersion.setSelected(Objects.requireNonNullElse(shouldIgnore, false));
+
+        final Boolean isShowAdditionalPath = settingsService.getHasAdditionalPath();
+        additionalPath.setEnabled(
+                Objects.requireNonNullElse(isShowAdditionalPath, false)
+        );
+
+        if (settingsService.getAdditionalPath() == null) {
+            final String basePath = Settings.getMagentoPath(project);
+
+            if (basePath != null) {
+                additionalPath.getComponent().setText(basePath);
```

Comment:
Why did you do that? What is business goal here?

Reply:
Hi @bohdan-harniuk , the functionality was duplicated as for the field modulePath so that you can specify an additional path

Reply:
Thanks for your code review. I fixed. Please check.